tests/cases/elements/flickable.slint

```diff
@@ -31,23 +31,23 @@ TestCase := Window {
 
 ```rust
 // Test that basic scrolling works, and that releasing the mouse animates
-use slint::{PointerEvent, PointerEventButton, euclid::point2};
+use slint::{WindowEvent, PointerEventButton, euclid::point2};
 let instance = TestCase::new();
-instance.window().dispatch_pointer_event(PointerEvent::Moved { position: point2(300.0, 100.0) });
+instance.window().dispatch_event(WindowEvent::PointerMoved { position: point2(300.0, 100.0) });
 slint::testing::mock_elapsed_time(5000);
-instance.window().dispatch_pointer_event(PointerEvent::Pressed { position: point2(300.0, 100.0), button: PointerEventButton::Left });
+instance.window().dispatch_event(WindowEvent::PointerPressed { position: point2(300.0, 100.0), button: WindowEventButton::Left });
 assert_eq!(instance.get_offset_x(), 0.);
 assert_eq!(instance.get_offset_y(), 0.);
-instance.window().dispatch_pointer_event(PointerEvent::Moved { position: point2(200.0, 50.0) });
+instance.window().dispatch_event(WindowEvent::PointerMoved { position: point2(200.0, 50.0) });
 assert_eq!(instance.get_offset_x(), 100.);
 assert_eq!(instance.get_offset_y(), 50.);
 slint::testing::mock_elapsed_time(200);
 assert_eq!(instance.get_offset_x(), 100.);
 assert_eq!(instance.get_offset_y(), 50.);
-instance.window().dispatch_pointer_event(PointerEvent::Moved { position: point2(100.0, 50.0) });
+instance.window().dispatch_event(WindowEvent::PointerMoved { position: point2(100.0, 50.0) });
 assert_eq!(instance.get_offset_x(), 200.);
 assert_eq!(instance.get_offset_y(), 50.);
-instance.window().dispatch_pointer_event(PointerEvent::Released { position: point2(100.0, 50.0), button: PointerEventButton::Left });
+instance.window().dispatch_event(WindowEvent::PointerReleased { position: point2(100.0, 50.0), button: WindowEventButton::Left });
```

Comment:
```suggestion
instance.window().dispatch_event(WindowEvent::PointerReleased { position: point2(100.0, 50.0), button: PointerEventButton::Left });
```

Reply:
Which raise the question of the name of this enum.
`PointerEventButton` is alright but since PointerEvent is not the name of the public enum, it is not so great. (But the name of the .slint enum must stay like that)
Should it be renamed to `WindowEventButton`? I guess that's not such a good name.
Maybe just  `PointerButton` ?

Reply:
I agree with `PointerButton` - that makes sense to me.

Reply:
Let's leave that for this PR: It is exposed to .slint files as part of the `PointerEvent` we pass around.